bin/oscapd-evaluate

```diff
@@ -165,17 +165,16 @@ def cli_scan(args, config):
                     if not args.no_standard_compliance:
                         es = evaluation_spec.EvaluationSpec()
                         es.mode = oscap_helpers.EvaluationMode.STANDARD_SCAN
-                        es.profile_id = args.profile
                         es.target = target
                         es.cpe_hints = cpes
                         ssg_sds = config.get_ssg_sds(cpes)
-                        profiles = oscap_helpers.get_profile_choices_for_input(ssg_sds, None)
-                        if es.profile_id not in profiles:
-                            raise RuntimeError(
-                                "Profile with id='%s' doesn't exist on target '%s'. "
-                                "Get the available profiles using\n"
-                                "$ oscapd-evaluate target-profiles --target %s"
-                                % (es.profile_id, target, target)
+                        try:
+                            args.profile = es.select_profile_by_suffix(ssg_sds, args.profile)
+                        except RuntimeError:
```

Comment:
this is quite dangerous, a lot of things can throw RuntimeError, please make this exception typed